server/templates/auxiliary/labels.html

```diff
@@ -0,0 +1,56 @@
+{#-
+Copyright 2024 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-#}
+
+<div id="metadata-labels" class="d-none">
+  <div data-label="Data Commons" data-value="{% trans %}Data Commons{% endtrans %}"></div>
```

Comment:
Make sure to include `{# TRANSLATORS #}` comments (see my other comment about this)